src/runtime_src/core/common/info_aie.cpp

```diff
@@ -70,10 +70,20 @@ addnodelist(const std::string& search_str, const std::string& node_str,
 static void
 populate_aie_dma(const boost::property_tree::ptree& pt, boost::property_tree::ptree& pt_dma)
 {
+  boost::property_tree::ptree fifo_len_pt;
   boost::property_tree::ptree mm2s_array;
   boost::property_tree::ptree s2mm_array;
   boost::property_tree::ptree empty_pt;
 
+  // Extract FIFO SIZE/COUNT information
+  auto fifo_len = pt.get_child("dma.fifo_len", empty_pt).begin();
+  boost::property_tree::ptree channel;
+  channel.put("fifo_size", fifo_len->second.data());
```

Comment:
This information shouldnt' be part of channel. This information should be part of only dma property tree.